voicevox_engine/app/routers/tts_pipeline.py

```diff
@@ -401,6 +402,74 @@ def multi_synthesis(
             background=BackgroundTask(try_delete_file, f.name),
         )
 
+    @router.post(
+        "/stream_synthesis",
+        response_class=StreamingResponse,
+        responses={
+            200: {
+                "content": {
+                    "audio/wav": {"schema": {"type": "string", "format": "binary"}}
+                },
+            }
+        },
+        tags=["音声合成"],
+        summary="ストリーミングで音声合成し、wavバイナリを逐次的に返す。24kHzモノラルのみ対応",
+    )
+    def stream_synthesis(
+        query: AudioQuery,
+        style_id: Annotated[StyleId, Query(alias="speaker")],
+        enable_interrogative_upspeak: Annotated[
+            bool,
+            Query(
+                description="疑問系のテキストが与えられたら語尾を自動調整する",
+            ),
+        ] = True,
+        core_version: str | SkipJsonSchema[None] = None,
+    ) -> StreamingResponse:
+        if query.outputSamplingRate != 24000:
+            raise HTTPException(
+                status_code=422,
+                detail="24kHz以外のサンプリングレートはサポートされていません",
+            )
+        if query.outputStereo:
+            raise HTTPException(
+                status_code=422,
+                detail="ステレオ出力はサポートされていません",
+            )
+        version = core_version or LATEST_VERSION
+        engine = tts_engines.get_engine(version)
```

Comment:
ここ、最新のmasterブランチに合わせるとこうなりそうです！

```suggestion
        engine = tts_engines.get_tts_engine(version)
```